contracts/src/token/erc20/extensions/metadata.rs

```diff
@@ -70,9 +67,6 @@ impl IErc20Metadata for Erc20Metadata {
     }
 
     fn decimals(&self) -> u8 {
-        // TODO: Use `U8` an avoid the conversion once
-        // https://github.com/OffchainLabs/stylus-sdk-rs/issues/117
-        // gets resolved.
```

Comment:
This was fixed, so we could change it, but do we actually want this?
- Since this is a view function and not intended to be put in storage, don't see a compelling reason to do so.
- It would involve removing the constant for `DEFAULT_DECIMALS` or converting in place to U8

Reply:
Do not see any valid reason tbh.